__tests__/actions/authActions.test.js

```diff
@@ -0,0 +1,132 @@
+import { wallet } from 'neon-js'
+
+import { wifLoginActions } from '../../app/actions/authActions'
+
+describe('authActions', () => {
+  describe('wifLoginActions', () => {
+    describe('request', () => {
+      const wif = 'KxB52D1FGe5xBn6YeezNwj7grhkHZxq7bv2tmaCPoT4rxApMwMvU'
+      const address = 'AM22coFfbe9N6omgL9ucFBLkeaMNg9TEyL'
```

Comment:
This address doesn't correspond to its WIF or private key. Intended?

Reply:
It doesn't matter, but it was not intended.  I'll update regardless.

Reply:
Fixed.